### ISSUE TITLE
Place Q&As Modal in the CMS is not available with keyboard

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10099]
## Describe the defect
On FAQ page content type (and soon the Resources and Support Detail page) the editor has the ability to add a Q&A Group with multiple Q&As.  If the user is tabbing through the form and activates the button for "Place Q&As" a modal opens that should allow the editor to choose which questions they would like to display in the group.  However, the keyboard focus is taken to the close button of the modal and the user is unable to tab to anything else in the modal, focus is stuck to the close button

## To Reproduce
1. Go to an FAQ page within the CMS. [Example page](https://staging.cms.va.gov/node/8233/edit)
2. Scroll down to Q&A Groups
3. Use the keyboard to tab to the "Place Q&As" button
4. Hit enter to activate the button and activate the modal
6. Hit tab to move through the modal
7. Notice that the green focus indicator is on the X to close the modal
8. Continue trying to tab through the modal and notice that the focus does not move from the X Close button

## AC / Expected behavior

- [x] When the modal opens, the user should be able to tab through all focusable elements within the modal
- [x] The user should not be able to tab to anything outside of/behind the modal
- [x] The modal should be able to be closed with the Close button and/or the ESC keyboard key.  Also in this case the "Select Q&As" button as that follows the use-case workflow
- [x] When the modal is closed (using any of the above scenarios), the focus should be taken back to the original trigger, in this case the "Place Q&As" button

## Screenshots
Place Q&As button
![image](https://user-images.githubusercontent.com/106678594/183504486-66f2b3c4-2501-4428-b12c-c9b59d90f0d5.png)

Modal with focus indicator on close button
![image](https://user-images.githubusercontent.com/106678594/183504623-9dc46141-9c74-44e0-82d5-66cf69b5cd72.png)

 ## Additional context
This is also happening on the Select Benefits Hubs modal further down in the same content type. I do not know if one solution would resolve both instances or if I should create a separate ticket for the Benefits Hubs modal.  

## Desktop (please complete the following information if relevant, or delete)
Issue is occurring on Chrome, FF and Edge

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by edmund-dunn** [comment]
This appears to be an issue with Drupal core and jquery-ui. See https://www.drupal.org/project/drupal/issues/3038336

I am attempting to identify a patch we can use to address it.

**Comment by EWashb** [comment]
@edmund-dunn that sounds like a plan! Thank you.

Also, can you add your point estimate to this ticket?

**Comment by edmund-dunn** [comment]
After discussing with @laflannery we will not be doing the focus back portion of this because it would take too much work to implement and this is a larger issue for Drupal core. The work I have done on this will need to be removed when we upgrade to D10.
